File: client/src/app/model/payment.ts
export interface Payment {
  paymentNo: string;
  paymentStatus: PaymentStatus;
  amount: string;
  vehicleId: number;
  bookingNo:string;
}


export enum PaymentStatus {
  // Define your payment status enum values here
  PENDING = 'PENDING',
  SUCCESSFUL = 'SUCCESSFUL',
  AMBIGIOUS = 'AMBIGIOUS'
}

File: src/main/java/com/project/vrs/postgres/model/Payment.java
package com.project.vrs.postgres.model;

import com.project.vrs.enums.PaymentStatus;
import com.project.vrs.shared.domain.AbstractEntity;
import jakarta.persistence.Entity;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.io.Serializable;

@EqualsAndHashCode(callSuper = true)
@Data
@Entity
public class Payment extends AbstractEntity implements Serializable {

    private String paymentNo;

    private PaymentStatus paymentStatus;

    private String amount;

    private Long vehicleId;

    private String bookingNo;
}
